src/itty-fetcher.spec.ts

```diff
@@ -131,6 +132,13 @@ describe('fetcher', () => {
         expected: { body: formdata },
       },
 
+      // Manual body property
+      'will pass Blob as-is (no stringify or content-type injection)': {
+        method: 'post',
+        payload: new Blob(['foo'], { type: 'text/plain' }),
+        expected: { headers: {} },
```

Comment:
Confused what this test is for? You should assert that a blob was passed in right?

Reply:
hmmm, yep - that's embarrassing! 😄